### PR TITLE
sync radarr to sonarr streaming CF

### PR DESCRIPTION
**Pull request by 8a8al00ey**
# Pull request

**Purpose**
sync radarr to sonarr streaming CF.

**Approach**
If this Pull Request is created to solve a issue explain how does this change address the problem?

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [x] Use github checklists. When solved, check the box and explain the answer.

**Learning**
If you're adding a new Custom Format make sure you follow the [Radarr/Sonarr Custom Format (JSON) Guidelines](https://github.com/TRaSH-/Guides/blob/master/.github/CONTRIBUTING.md).

**Requirements**
Check all boxes as they are completed

- [X] These changes meet the standards for [contributing](https://github.com/TRaSH-/Guides/blob/master/.github/CONTRIBUTING.md).
- [X] I have read the [code of conduct](https://github.com/TRaSH-/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by TRaSH- — COMMENTED** [review]
@FonduemangVI didn't we need to change something with the disney+ one ?

File: docs/json/sonarr/cf/hbo.json [hunk 1]
```diff
@@ -8,9 +8,27 @@
       "name": "HBO",
       "implementation": "ReleaseTitleSpecification",
       "negate": false,
+      "required": true,
+      "fields": {
+        "value": "\\b(hbo)(?![-_. ]max)\\b"
```
Comment (8a8al00ey) [inline docs/json/sonarr/cf/hbo.json:13]: Tried to account for not matching hbo max

**Comment by FonduemangVI** [comment]
> @FonduemangVI didn't we need to change something with the disney+ one ?

We did yes. Not sure if we should be matching the Sonarr to Radarr or Radarr to Sonarr CFs. Need to test that they will import and work though

**Comment by bakerboy448** [comment]
Radarr is what _should_ be used

**Comment by FonduemangVI** [comment]
@TRaSH- this DSNP is fine because it's taking the source out which is probably the better way to do it as Baker said

**Pull request merged**